scenario_player/node_support.py

```diff
@@ -343,6 +347,15 @@ def _command(self):
                 else:
                     cmd.append(f"--{option_name}")
```

Comment:
Doesn't this already handle positive flags properly? It looks like we could replace this PR by
```python
FLAG_OPTIONS += {'no-' + opt for opt in FLAG_OPTIONS}
```
so that negative options are properly handled, too.

Reply:
Ach, I got `FLAG_OPTIONS` confused with the pre-defined lists in the `MANAGED_*` constants. 
You are correct, that would work just fine.